src/components/ScorecardCategory.tsx

```diff
@@ -116,62 +104,56 @@ function ScorecardCategory({
       dense
       onClick={toggleChecked}
     >
-      <ListItemText>
```

Comment:
Github is very confused, but all that changed here is that the checkbox is small now and this isn't inside a text wrapper.